test/specs/components/action.spec.jsx

```diff
@@ -195,19 +221,24 @@ xdescribe('Action', () => {
         });
     });
 
-    describe('buy action without stripe keys', () => {
+    // Re-visit this test after fixing link fallback
+    xdescribe('buy action without stripe keys', () => {
         const props = getBuyProps();
-
-        beforeEach(() => {
-            mockedStore = mockAppStore(sandbox, getStoreState({
-                app: {
-                    publicKeys: {
-                        stripe: false
-                    }
+        const context = getContext({
+            app: {
```

Comment:
can you nest all the tests that require stripe under one `describe` and do this setup in a higher-level `beforeEach`?